### ISSUE TITLE
Screen reader improvements

### ISSUE THREAD

**Issue by lohnn** [lohnn/lohnn_podcast#61]
There are some places where the app is not accessible enough through screen readers.
This will be a parent issue where we will declare the issues separately in separate sub-issues.
